docs/streams/quickstart.html

```diff
@@ -131,18 +140,12 @@
     --partitions 1 \
     --topic streams-plaintext-input
 Created topic "streams-plaintext-input".
-</pre>
 
-Note: we create the output topic with compaction enabled because the output stream is a changelog stream
-(cf. <a href="#anchor-changelog-output">explanation of application output</a> below).
-
-<pre class="brush: bash;">
 &gt; bin/kafka-topics.sh --create \
     --zookeeper localhost:2181 \
     --replication-factor 1 \
     --partitions 1 \
-    --topic streams-wordcount-output \
```

Comment:
I think all the code-related changes introduced in this page are regressions actually.